docs/Reference/web3js-eea-Methods.md

```diff
@@ -0,0 +1,144 @@
+description: web3js-eea methods reference
+<!--- END of page meta data -->
+
+Use the [web3.js-eea library](https://github.com/PegaSysEng/eeajs) to [create and send 
+private transactions](../Privacy/Private-Transactions/eeajs.md).
+
+## Options Parameter 
+
+The Options parameter is used by: 
```

Comment:
Is "Options" the actual parameter name? If so, should it be formatted?

Reply:
No - not an actual parameter